torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -261,15 +261,24 @@ struct TORCH_CUDA_API TensorView : public Val {
   // Reorder axes according to old2new[old_pos] = new_pos
   TensorView* reorder(const std::unordered_map<int, int>& old2new);
 
-  /*
-   * WARNING: Does not return this TensorView, returns a new tensorview consumed
-   * to create this!! Take reduction axes out of this domain, and create a new
-   * domain. New domain will be used to create this domain. For example: TV1[I0,
-   * I1] = TV0[I0, R0, R1, I1] TV0->rfactor({1}) TV0 is transformed to ->
-   * TV0[I0, R1, I1] The TensorView returned is: TV2[I0, R0, I3, I1] The
-   * reduction will now beset as: TV1[I0, R1, I1] = TV2[I0, R0, I3, I1] TV0[I0,
-   * I1] = TV1[I0, R1, I1]
-   */
+  // WARNING: Does not return this TensorView, returns a new tensorview consumed
```

Comment:
@csarofeen : please review this

Reply:
nit: `consumed by this` (I know it's my own English I'm nit picking.)